util/src/main/resources/SummaryReport.ftl

```diff
@@ -236,6 +236,8 @@
         <h2>
           <a name="places" href="#places">Series Summaries for Sample Places</a>
         </h2>
+        <button onclick="open_all_details()">Expand All</button>
+        <button onclick="close_all_details()">Collapse All</button>
```

Comment:
Wonder about putting it under TOC? And have that apply to everything under TOC and later sections.